.rubocop.yml

```diff
@@ -1,8 +1,10 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
+
```

Comment:
Fix all obsolete cops name and remove disable cops.
We can remove `rubocop-rails` as it is not used

Reply:
Fixed bc98e21e